pyproject.toml

```diff
@@ -16,7 +16,7 @@ requires-python = ">=3.7"
 dependencies = [
     "apache-airflow>=2.0",
     "markupsafe>=1.1.1,<2.1.0",
```

Comment:
Worth investigating in a separate PR/issue if we can remove `markupsafe`

Reply:
@kaxil I'll create a ticket for the same. Should it be part of this sprint?

Reply:
Yes please